Firebase/Auth/Source/Public/FIRAuth.h

```diff
@@ -288,14 +288,15 @@ NS_SWIFT_NAME(FIRActionCodeURL)
 @property(nonatomic, nullable, copy, readonly) NSString *languageCode;
 
 /** @fn actionCodeURLWithLink:
-    @brief Construct an FIRActionCodeURL from an oob link.
+    @brief Construct an FIRActionCodeURL from an out of band link (e.g. email link).
     @param link The oob link string used to construct the action code URL.
     @return The FIRActionCodeURL object constructed based on the oob link provided.
  */
-+ (nullable instancetype)actionCodeURLWithLink:(NSString *)link;
++ (nullable instancetype)actionCodeURLWithLink:(NSString *)link
+    NS_SWIFT_NAME(actionCodeURL(link:));
```

Comment:
Yeah, verified it's auto translated.

Reply:
Just ton confirm... when the `actionCodeURL(link:)` is there this will turn into:
`ActionCodeURL.actionCodeURL(link:)` but that's not what the API in Swift should be.

We shouldn't specifically add the `NS_SWIFT_NAME` here and the API should turn into: `init(link:)` which can be used like `ActionCodeURL(link: "http://....")`

Reply:
to `init(link:)`
just pushed the changes.